conda/gateways/disk/create.py

```diff
@@ -246,3 +254,53 @@ def compile_pyc(python_exe_full_path, py_full_path):
                   "  stderr: %s\n"
                   "  rc: %d", command, stdout, stderr, rc)
         raise RuntimeError()
+
+
+def get_json_content(path_to_json):
+    if isfile(path_to_json):
+        try:
+            with open(path_to_json, "r") as f:
+                json_content = json.load(f)
+        except json.decoder.JSONDecodeError:
+            json_content = {}
+    else:
+        json_content = {}
+    return json_content
+
+
+def create_private_envs_meta(pkg, prefix):
+    # type: (str, str -> ()
+    path_to_conda_meta = join(context.root_prefix, "conda-meta")
+    path_to_private_envs = join(path_to_conda_meta, "private_envs")
```

Comment:
I originally had it as `private_envs.json` but there is some code somewhere that (I think having to do with `linked_data`) that scans `conda-meta` for files ending in .json to generate a list of packages that are installed. So having the .json extension on the file was messing it up. So, I would suggest any file name that does not end in .json